modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js

```diff
@@ -419,20 +425,20 @@ AUI.add(
 
 								item.attr('checked', '');
 								item.attr('name', newName);
-								item.attr('id', newName);
+								item.attr('id', newId);
 							}
 							else if (
 								inputNodeName === 'button' ||
 								inputNodeName === 'div' ||
 								inputNodeName === 'span'
 							) {
 								if (oldName) {
-									item.attr('id', newName);
+									item.attr('id', newId);
```

Comment:
Shall we check `oldId` here? checking for `oldName` then setting with `newId` looks off

Reply:
Yes, it makes sense. I'll send an extra commit making this change.